extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -212,6 +212,23 @@ function tests(name) {
       })).to.be.not.be.rejected;
     });
 
+    it('should only layout once', () => {
+      return getAd({
+        width: 300,
+        height: 250,
+        type: '_ping_',
+        src: 'testsrc',
```

Comment:
Seems many test don't care about the param you pass in here, we can just extract another layer of helper method to save some more lines.